spec/game_spec.rb

```diff
@@ -0,0 +1,150 @@
+require_relative '../lib/game'
+
+describe Game do
+  describe '#add' do
+    subject(:game) { described_class.new }
```

Comment:
Notice your `subject` is the same all across these examples. You can move it up a level and reuse between all these tests